Packs/CloudIncidentResponse/Playbooks/playbook-Cortex_XDR_-_XCloud_Token_Theft_Response.yml

```diff
@@ -0,0 +1,2269 @@
+      Azure-resourceRemediationType:
+        simple: Poweroff
+      Azure-userRemediationType:
+        simple: Disable
+      GCP-resourceRemediationType:
+        simple: Stop
+      GCP-userRemediationType:
+        simple: Disable
+      accessKeyId:
+        complex:
+          root: CloudIndicators
+          accessor: access_key_id
+      autoAccessKeyRemediation:
+        simple: "False"
+      autoBlockIndicators:
+        simple: "True"
+      autoResourceRemediation:
+        simple: "False"
+      autoUserRemediation:
+        simple: "True"
+      cloudProvider:
+        complex:
+          root: incident.xdralerts
+          accessor: cloudprovider
+          transformers:
+          - operator: uniq
+      username:
+        complex:
+          root: CloudIndicators
+          accessor: username
+    separatecontext: false
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 890,
+          "y": 3005
+        }
+      }
+    note: false
+    timertriggers:
+    - fieldname: containmentsla
+      action: start
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "52":
+    id: "52"
+    taskid: 3355a9b9-c68a-4851-8dd3-9020c4e35043
+    type: condition
+    task:
+      id: 3355a9b9-c68a-4851-8dd3-9020c4e35043
+      version: -1
+      name: Check the VPN list type
+      description: Checks if the provided data is comma separated or an URL.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "55"
+      URL:
+      - "53"
+    separatecontext: false
+    conditions:
+    - label: URL
+      condition:
+      - - operator: startWith
+          left:
+            value:
+              complex:
+                root: inputs.VPNIPList
+            iscontext: true
+          right:
+            value:
+              simple: http://
+          ignorecase: true
+        - operator: startWith
+          left:
+            value:
+              complex:
+                root: inputs.VPNIPList
+            iscontext: true
+          right:
+            value:
+              simple: https://
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -630,
+          "y": -720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "53":
+    id: "53"
+    taskid: 6d1c856f-4737-4871-8144-a2190b328b79
+    type: regular
+    task:
+      id: 6d1c856f-4737-4871-8144-a2190b328b79
+      version: -1
+      name: Process the VPN IP list
+      description: This script will extract indicators from given HTML and will handle bad top-level domains to avoid false positives caused by file extensions.
```

Comment:
```suggestion
      description: This script will extract indicators from a given HTML and will handle bad top-level domains to avoid false positives caused by file extensions.
```